src/components/games/GameItem.js

```diff
@@ -51,14 +51,16 @@ const GameItem = (props) => {
   return (
     <li className={classes.list}>
```

Comment:
I can see you're using li here, but I don't see this wrapped in ul:
```
const GamesList = (props) => {
  return props.games.map((game) => (
    <GameItem
      homeTeamScore={game.home_team_score}
      visitorTeamScore={game.visitor_team_score}
      homeTeamId={game.home_team.id}
      visitorTeamId={game.visitor_team.id}
      date={game.date}
      key={game.id}
      id={game.id}
    />
  ));
};
```

you should wrap the GameList return in a ul/ol tag.

Reply:
the ul tag is in the parent component: TeamGamesPage.js.
Do you think it's confusing or bad practice?